lib/src/actions.dart

```diff
@@ -59,13 +61,25 @@ class CustomSlidableAction extends StatelessWidget {
   /// {@endtemplate}
   final bool autoClose;
 
-  /// {@template slidable.actions.onPressed}
-  /// Called when the action is tapped or otherwise activated.
+  /// {@template slidable.actions.onpressed}
+  /// called when the action is tapped or otherwise activated.
   ///
-  /// If this callback is null, then the action will be disabled.
+  /// if this callback is null, then the action will be disabled.
```

Comment:
These capitalization changes looks like mistakes?

Reply:
yeap 😅 sorry for that